mdformat_admon/plugin.py

```diff
@@ -29,14 +29,23 @@ def _render_admon(node: RenderTreeNode, context: RenderContext) -> str:
     separator = "\n\n"  # TODO: Is this configurable?
     indent = " " * 4  # FIXME: Is this configurable?
     title = node.info.strip()
-    body = f"{node.markup} {title}{separator}"
+    title_line = f"!!! {title}"
```

Comment:
I think the `_render_admon_title` should be called here since admonitions can also start with `...` (https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions), but I'll make those changes, if any in a separate PR

Reply:
> since admonitions can also start with ...

 Do you know if Python-Markdown allows this? https://python-markdown.github.io/extensions/admonition/ doesn't seem to mention the ... syntax. What about the markdown-it parser plugin?

Anyways, I think it'd be more in the spirit of mdformat to unify the starting sequence, but this is, of course, your plugin and you're free to decide otherwise :smile_cat:

Reply:
I'm not sure the rationale around what is and isn't supported, but my use case is for `mkdocs`, so I'll probably revisit the plugin and try to expand support for at least the `???` and `???+` syntax

Reply:
> Admonitions seems to be the biggest controversy in Markdown since sliced bread

Yep. Probably best to come up with a clear spec for this plugin (and the parser extension), e.g. only support `!!!` or only support whatever mkdocs supports.